Library/Formula/jq.rb

```diff
@@ -22,7 +22,7 @@ class Jq < Formula
   depends_on "bison" => :build # jq depends on bison > 2.5
 
   def install
-    system "autoreconf", "-iv"
+    system "autoreconf", "-iv" if build.devel? || build.head?
```

Comment:
Haven't changed the `if` condition to reflect the removal of `devel` block as wouldn't matter, and might be required in future. But yes, `build.devel?` is redundant here. Should I remove it?

Yes, `jq` builds with `autoconf`, `automake` and `libtool` in `head` block.